1/test/AddTodo.spec.js

```diff
@@ -0,0 +1,9 @@
+import {mount} from '@vue/test-utils'
+import AddTodo from '../src/components/AddTodo.vue'
```

Comment:
Try to put the test file next to the file with the implementation.

Instead of:
```sh
$ tree src test
src
├── App.vue
├── components
│   ├── AddTodo.vue
│   ├── layout
│   │   └── Header.vue
│   ├── TodoItem.vue
│   └── Todos.vue
└── main.js
test
├── AddTodo.spec.js
├── App.spec.js
├── TodoItem.spec.js
└── Todos.spec.js
```

You would have:

```sh
$ tree src
src
├── App.spec.js
├── App.vue
├── components
│   ├── AddTodo.spec.js
│   ├── AddTodo.vue
│   ├── layout
│   │   └── Header.vue
│   ├── TodoItem.spec.js
│   ├── TodoItem.vue
│   ├── Todos.spec.js
│   └── Todos.vue
└── main.js
```